packages/teleterm/src/services/config/providers/appearanceConfigProvider.ts

```diff
@@ -0,0 +1,33 @@
+import { ConfigServiceProvider } from '../types';
+
+export type AppearanceConfig = {
+  fonts: {
+    proportionalFontFamily: string;
+    monospacedFontFamily: string;
+  };
+};
+
+export const appearanceConfigProvider: ConfigServiceProvider<AppearanceConfig> =
+  {
+    provideDefaultsForPlatform(platform) {
```

Comment:
Yes, it is, because we have type for the whole object
<img width="586" alt="image" src="https://user-images.githubusercontent.com/20583051/147927445-1d539787-b020-4c23-93d7-34eb5b01b350.png">